assets/js/blocks/attribute-filter/block.js

```diff
@@ -227,6 +220,13 @@ const AttributeFilterBlock = ( {
 		]
 	);
 
+	// Track checked STATE changes - if state changes, update the query.
+	useEffect( () => {
+		if ( ! blockAttributes.showFilterButton ) {
+			onSubmit( checked );
+		}
+	}, [ blockAttributes.showFilterButton, checked, onSubmit ] );
```

Comment:
Odd that this created a changeset. I assume it was spacing changes maybe?

Reply:
This was using `onSubmit ` before it was defined. I had an error when testing a build that didn't support ie11. We'll need this change in place at some point.

Reply:
ahhh! I missed the line number change.